src/features/user/ManageProjects/components/ProjectMedia.tsx

```diff
@@ -115,14 +115,16 @@ export default function ProjectMedia({
   const [files, setFiles] = React.useState([]);
 
```

Comment:
Is `files` used anymore? If not, this can be removed. Remove code that references `files` as well.